hrjobcontract/js/test/controllers/modal/modal-contract_new_test.js

```diff
@@ -0,0 +1,120 @@
+define([
```

Comment:
the name of this file is wrong, i think. shouldn't it be `modal-contract-new_test.js`